common/app/model/meta.scala

```diff
@@ -701,7 +701,6 @@ final case class Tags(
   private val isLabourLiverpool = tags.exists(t => t.id == "membership/series/labour-liverpool")
   private val isViewFromMiddleTown = tags.exists(t => t.id == "membership/series/election-2016-the-view-from-middletown")
   private val isNewRetirement = tags.exists(t => t.id == "membership/series/the-new-retirement")
-  lazy val isExploreSeries = isLabourLiverpool || isViewFromMiddleTown || isNewRetirement
```

Comment:
We can probably kill all `isLabourLiverpool `, `isViewFromMiddleTown` and `isNewRetirement` too!